org_status/org_status.py

```diff
@@ -83,18 +85,32 @@ def main():
 
     styled = (lambda l, *_: l) if args.no_color else colored
     verbose = print if args.verbose else (lambda *_: None)
+    hosts_only_print = print if args.hosts_only else verbose
```

Comment:
Interesting.

Reply:
Should I remove this? Any thoughts on how it could be better?